src/transformStateWithClones.js

```diff
@@ -1,13 +1,41 @@
 'use strict';
 
 /**
- * @param {Object} state
+ * @param {Object} newState
  * @param {Object[]} actions
  *
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newState = { ...state };
+  const returnObjects = [];
```

Comment:
this name looks like a name of a function to me.
When we name a function - its name should describe what it does, so it should be a verb
When we name a variable - it should describe **what** is stored inside, so it is better to be a noun (except for boolean variables, they start with ...should or sth like that)

```suggestion
  const objectsToReturn = [];
```
If we want to do it even better - it should be
```suggestion
  const stateVersions = [];
```

Reply:
I understand, will change it here and check other tasks. Thank's for such wide advise!

Reply:
named them stateClones